manifest.toml

```diff
@@ -30,8 +30,6 @@ ram.runtime = "500M"
 [install]
   [install.domain]
   type = "domain"
-  ask.en = "The domain name to use. Bonfire will be installed at its root path (/). This can't be changed."
```

Comment:
Why removing this ?

Reply:
well... according to the linter warning, that should be removed

Reply:
But then we add no help ?
The information about the root path and the impossibility to change that path afterwards is very important in my opinion.
I prefer to keep it if it makes no trouble to the package.

(I'm trying to setup a working app, linter warnings can wait)

Reply:
this is no `help`, this is `ask` and `ask` is obsolete

Reply:
So I should change it to help ?

Reply:
yes